### PR TITLE
(1681) Rolled up forecasts must be calculated in IATI XML

### PR DESCRIPTION
**Pull request by CristinaRO**
## Changes in this PR
- Sum programme and child activity forecasts by financial quarter in IATI XML

## Next steps

- [ ] Is an ADR required? An ADR should be added if this PR introduces a change to the architecture.
- [x] Is a changelog entry required? An entry should always be made in `CHANGELOG.md`, unless this PR is a small tweak which has no impact outside the development team.
- [ ] Do any environment variables need amending or adding?
- [ ] Have any changes to the XML been checked with the IATI validator? See [XML Validation](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/doc/xml-validation.md)

### CODE REVIEW

**Comment by CristinaRO** [comment]
['Rolled up' forecasts must be calculated and included at level B in the IATI xml](https://trello.com/c/768kwm57/1681-rolled-up-forecasts-must-be-calculated-and-included-at-level-b-in-the-iati-xml)

File: app/models/financial_quarter.rb [hunk 1]
```diff
@@ -28,6 +30,14 @@ def for_date(date)
     end
   end
 
+  def <=>(other)
```
Comment (pezholio) [inline app/models/financial_quarter.rb:33]: 👍

**Review by pezholio — APPROVED** [review]
Great stuff, all makes sense. Let's 🚢 it! 👍

**Comment by CristinaRO** [comment]
Thank you @pezholio !

**Pull request merged**